src/app/shared/components/provider-info/provider-info.component.ts

```diff
@@ -44,21 +43,6 @@ export class ProviderInfoComponent implements OnInit {
 
   constructor(private store: Store) {}
 
-  ngOnInit(): void {
-    this.store.dispatch(new GetInstitutionStatus());
-    this.institutionStatuses$
-      .pipe(
-        filter((institutionStatuses) => !!institutionStatuses.length),
-        takeUntil(this.destroy$)
```

Comment:
revert this change please, this field is still needed. 
![image](https://user-images.githubusercontent.com/27493959/172439547-212f1e3b-623b-425c-a56b-2768be2d33cc.png)
![image](https://user-images.githubusercontent.com/27493959/172439074-59d30b2a-7223-4346-9bad-9f097debbf0b.png)